src/pages/timer.js

```diff
@@ -42,62 +44,81 @@ function Timer() {
 
 		const interval = setInterval(() => getTime(now), 1000);
 		return () => clearInterval(interval);
-	}, [seconds, minutes, hours, days]);
+	}, [seconds, minutes, hours]);
 
 	return (
 		<>
-			<Layout>
-				<CookingTimer
-					timerDays={timerDays}
-					timerHours={timerHours}
-					timerMinutes={timerMinutes}
-					timerSeconds={timerSeconds}
-				/>
-				<form
-					onSubmit={event => {
-						event.preventDefault();
-						setUserInputDays('');
-					}}
-				>
-					<label>Days:</label>
-					<StyledInputTime
-						type="text"
-						size="20"
-						maxLength="2"
-						value={userInputDays}
-						onChange={event => {
-							setUserInputDays(event.target.value);
-						}}
-					/>
-					<label>Hours:</label>
-					<StyledInputTime
-						type="text"
-						maxLength="2"
-						value={userInputHours}
-						onChange={event => {
-							setUserInputHours(event.target.value);
-						}}
-					/>
-					<label>Minutes:</label>
-					<StyledInputTime
-						type="text"
-						maxLength="2"
-						value={userInputMinutes}
-						onChange={event => {
-							setUserInputMinutes(event.target.value);
-						}}
-					/>
-					<label>Seconds:</label>
-					<StyledInputTime
-						type="text"
-						maxLength="2"
-						value={userInputSeconds}
-						onChange={event => {
-							setUserInputSeconds(event.target.value);
-						}}
-					/>
-				</form>
-			</Layout>
+			<StyledNavUl>
+				<StyledNavbar as="a" href="/">
+					Home
+				</StyledNavbar>
+
+				<StyledNavbar as="a" href="/timer">
+					Timer
+				</StyledNavbar>
+				<StyledNavbar as="a" href="/trackfood">
+					Food Tracker
+				</StyledNavbar>
+			</StyledNavUl>
+			<CookingTimer
+				timerDays={timerDays}
+				timerHours={timerHours}
+				timerMinutes={timerMinutes}
+				timerSeconds={timerSeconds}
+			/>
+			<form
+				onSubmit={event => {
+					event.preventDefault();
+				}}
```

Comment:
ther eis no functionality on submit implemented. In that case you dont need it :)